web/blog/2022-06-15-jobs-feature-announcement.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Feature Release Announcement - Wasp Jobs
+authors: [shayneczyzewski]
+image: /img/waspleau-screenshot.png
```

Comment:
is this a leftover from the prev post?

Reply:
Ah haha yeah thanks, great catch. I copied that as my starting point and didn't update it. Thanks, will do! 👍🏻

Reply:
Updated to the Jobs DSL snippet image!